ceph-releases/ALL/debian/daemon/__DOCKERFILE_INSTALL__

```diff
@@ -1,4 +1,5 @@
 echo 'Install packages' && \
+      DEBIAN_FRONTEND=noninteractive apt-get dist-upgrade && \
```

Comment:
This should probably be after the `apt-get update` that that can fetch an up to date list if the available packages

Reply:
ohh yes. Fixed.